group_members.go

```diff
@@ -36,12 +35,12 @@ type GroupMembersService struct {
 type GroupMember struct {
 	ID          int              `json:"id"`
 	Username    string           `json:"username"`
-	Email       string           `json:"email"`
 	Name        string           `json:"name"`
 	State       string           `json:"state"`
-	CreatedAt   *time.Time       `json:"created_at"`
 	AccessLevel AccessLevelValue `json:"access_level"`
 	ExpiresAt   *ISOTime         `json:"expires_at"`
+	AvatarURL   string           `json:"avatar_url"`
+	WebURL      string           `json:"web_url"`
```

Comment:
One small question... Could you reorder the last four lines so they match the order in the docs? I'm trying to follow the docs ordering a bit so it's easier to spot missing or new entries over time. So that would be:

```
	AvatarURL   string           `json:"avatar_url"`
	WebURL      string           `json:"web_url"`
	ExpiresAt   *ISOTime         `json:"expires_at"`
	AccessLevel AccessLevelValue `json:"access_level"`
```

Reply:
Yep. Absolutely. I will do that.